### PR TITLE
easier contributions.

### PR DESCRIPTION
**Pull request by nisarhassan12**
Hello. :slightly_smiling_face: 

This Pr makes contributions easier by fully automating the dev setup with Gitpod(a full-featured online IDE) which is free for OSS. With one click it'll start a workspace and automatically:

- clone the `vue-class-component` repo.
- install the dependencies.
- run `yarn dev`.
- run `yarn docs:dev`.

so that anyone interested in contributing can start straight away without any friction.

You can give it a try on my fork of the repo via the following link:

https://gitpod.io/#https://github.com/nisarhassan12/vue-class-component

This is how it looks:

image

### CODE REVIEW

**Comment by ktsn** [comment]
Thanks for the PR. Is there a way to analyze how many people used Gitpod for Vue Class Component contribution in a certain period? I'm a bit worry about this setting would be stale in future by no one uses it. So I want to assess how effective the tool is for contribution.

**Comment by nisarhassan12** [comment]
Thanks! @ktsn 

A lot of popular OSS projects out there are already using it simplify contributors onboarding some of them are:

- https://github.com/freeCodeCamp/freeCodeCamp
- https://github.com/facebook/docusaurus
- https://github.com/mozilla/pdf.js/

You can find a brief list of them at https://contribute.dev

image

**Comment by ktsn** [comment]
I mean I want to see some sort of usage report on this repository if I introduce it in this repo such as "how many people tried the code on Gitpod?", "Is this PR made with Gitpod?" etc.

**Comment by nisarhassan12** [comment]
Yes. this Pr is made with Gitpod so far it is very little but once the Pr is merged we'll add the project to https://contribute.dev/ which has a list of cool OSS projects that have automated their setup with Gitpod and people can start contributing to them in a single click and also when it'll be seen in the project's Readme and from there people may try it.

```
mysql> select count(distinct ownerid) users, count(id) workspaces, contextUrl from d_b_workspace where type = 'regular' and contextUrl like '%vue-class-component%' group by contextUrl orderby users desc limit 300;
+-------+------------+-----------------------------------------------------------------------+
| users | workspaces | contextUrl                                                            |
+-------+------------+-----------------------------------------------------------------------+
|     2 |          5 | https://github.com/nisarhassan12/vue-class-component                  |
|     1 |          1 | https://github.com/vuejs/vue-class-component                          |
|     1 |          1 | https://github.com/vuejs/vue-class-component/blob/master/src/index.ts |
+-------+------------+-----------------------------------------------------------------------+
3 rows in set (1.47 sec)

```

**Comment by ktsn** [comment]
I don't think we are in synced... I don't mean how vue-class-component appears in gitpod.

Again, I want to assess how people use Gitpod **for vue-class-component** once we enable this. I would be asking if I can see some sort of **usage report / analytics** for vue-class-component in Gitpod. Because we cannot evaluate if it's efficient for contribution without analyzing with actual usage data.

**Comment by nisarhassan12** [comment]
> Again, I want to assess how people use Gitpod for vue-class-component once we enable this. I would be asking if I can see some sort of usage report / analytics for vue-class-component in Gitpod. Because we cannot evaluate if it's efficient for contribution without analyzing with actual usage data.

Thanks. This makes sense. We are happy to share data on how many users created workspaces for vue-class-component, and how much time they've spent in total working on vue-class-component.In the future, we would like to share this data more publicly, e.g. by building a dashboard with stats about repositories & programming languages in Gitpod, but this is not implemented yet, so for now we have to query this data manually and send it to you. Please ping either me or @jankeromnes if you would like to get the latest data on vue-class-component / Gitpod usage, and we'll try to reply as soon as possible.

**Comment by ktsn** [comment]
Great. Thank you for describing about it. I'm looking forward to seeing dashboard 👍

File: README.md [hunk 1]
```diff
@@ -2,12 +2,18 @@
...
-[![npm](https://img.shields.io/npm/v/vue-class-component.svg)](https://www.npmjs.com/package/vue-class-component)
+[![npm](https://img.shields.io/npm/v/vue-class-component.svg)](https://www.npmjs.com/package/vue-class-component) [![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/vuejs/vue-class-component) 
 
 ## Document
 
 See [https://class-component.vuejs.org](https://class-component.vuejs.org)
 
+## Online one-click setup for contributing
+
+Contribute to Blaze using a fully featured online development environment that will automatically: clone the repo, install the dependencies and start the docs web server and run `yarn dev`.
```
Comment (ktsn) [inline README.md:13]:
```suggestion
Contribute to Vue Class Component using a fully featured online development environment that will automatically: clone the repo, install the dependencies and start the docs web server and run `yarn dev`.
```

Forgot to replace the package name?

**Pull request merged**